core/src/main/java/gyro/core/command/UpCommand.java

```diff
@@ -50,6 +56,20 @@ public void doExecute(RootScope current, RootScope pending, State state) {
                 break;
             }
 
+            if (!auditStarted) {
+                Map<String, Object> log = new HashMap<>();
+                log.put("commandArguments", getUnparsedArguments());
+
+                try {
+                    log.put("version", VersionCommand.getCurrentVersion().toString());
+                } catch (IOException e) {
+                    // Do nothing.
+                }
+
+                ui.startAuditors(log);
+                auditStarted = true;
+            }
+
```

Comment:
Might be nice to have this in a helper method so commands that want to audit don't have to copy this code.